packages/botonic-plugin-google-analytics/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@botonic/plugin-google-analytics",
+  "version": "0.14.0",
```

Comment:
only use 0.14.0 if you're going to publish it after merge. Otherwise, 0.14.0-rc1

Reply:
I think it will be published as it's needed for a project, so it's okay that it is `0.14.0`.